common/persistence/tests/cassandra_test.go

```diff
@@ -48,103 +49,87 @@ const (
 	testCassandraDatabaseNameSuffix = "temporal_persistence"
 )
 
-func TestCassandraExecutionMutableStateStoreSuite(t *testing.T) {
-	cfg := newCassandraConfig()
-	SetUpCassandraDatabase(cfg)
-	SetUpCassandraSchema(cfg)
-	logger := log.NewNoopLogger()
-	factory := cassandra.NewFactory(
-		*cfg,
+type cassandraTestData struct {
+	cfg *config.Cassandra
+	factory *cassandra.Factory
+	logger log.Logger
+}
+
+func setUpCassandraTest(t *testing.T) (cassandraTestData, func()) {
+	var td cassandraTestData
```

Comment:
nit: you use the local var "testData" in other places, "td" here ... consistency would be nice.

Reply:
Done